cookbooks/nginx_keep/recipes/default.rb

```diff
@@ -0,0 +1,31 @@
+if ['app_master','app','solo'].include? node[:instance_role]
+
+  template "/etc/nginx/servers/md2/custom.conf" do
```

Comment:
Can the app_name (md2 here) be specified as a variable?  
Like defining at the beginning of the file something like `app_name="myapp"` and then referencing it when needed.